CHANGELOG.md

```diff
@@ -7,24 +7,13 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [unreleased]
 
-### Changed
-
--   Adds unit tests to make sure that recipe functions are exported correctly
-
-## [0.24.4] - 2022-08-07
-
--   Fixes styling issue for login with github
+### Bug fixes
```

Comment:
There is already a section called Fixes for this release. Please add this comment in there.

Reply:
Done.